corehq/apps/enterprise/api/resources.py

```diff
@@ -388,3 +388,33 @@ def dehydrate(self, bundle):
 
     def get_primary_keys(self):
         return ('form_id', 'submitted',)
+
+
+class DataExportReportResource(ODataEnterpriseReportResource):
+    domain = fields.CharField()
+    name = fields.CharField()
+    export_type = fields.CharField()
+    export_subtype = fields.CharField()
+    owner = fields.CharField()
+
+    REPORT_SLUG = EnterpriseReport.DATA_EXPORTS
+
+    def get_report_task(self, request):
+        account = BillingAccount.get_account_by_domain(request.domain)
+        return generate_enterprise_report.s(
+            self.REPORT_SLUG,
+            account.id,
+            request.couch_user.username
+        )
+
+    def dehydrate(self, bundle):
+        bundle.data['domain'] = bundle.obj[0]
+        bundle.data['name'] = bundle.obj[1]
+        bundle.data['export_type'] = bundle.obj[2]
+        bundle.data['export_subtype'] = bundle.obj[3]
+        bundle.data['owner'] = bundle.obj[4]
+
+        return bundle
+
+    def get_primary_keys(self):
+        return ('domain', 'export_type', 'export_subtype', 'name')
```

Comment:
I tested on staging, I can have two export in same domain, with same export_type, export_subtype and name. In this case, adding owner won't be helpful. Should we add some other attribute like export_id?